### DISCUSSION TITLE
Delete Request != Removal Radarr

### DISCUSSION THREAD

**Discussion by ezekieldas** [seerr-team/seerr#729] [Q&A] [answered]
I just did a few quick tests and found removal of an item from Jellyseer doesn't remove the item from Radarr (and presumably Sonarr). I don't see a setting for this and assume this would be the expected behavior.

**Comment by fallenbagel** [comment]
Deleting request is not the same as deleting it from radarr...

You need to use the delete item button in the slide over modal on the item itself.

**Comment by ezekieldas** [comment]
Thanks for the suggestion! I'm not familiar with the terminology 'slide over modal.'

If I have a full view of the content/item (eg, http://localhost/movie/823464), there's a gear icon in the upper right marked 'Manage Movie'. Opening this reveals a right panel with 'delete request' but that too only removes it from the JS request queue, not the radarr queue.  Is this 'slide over modal' somewhere else?

**Reply by fallenbagel** [reply, marked as answer]
Once again. Delete request is not the same delete button as delete from radarr.

![Screenshot_20240426_202828_Chrome Dev.jpg](https://github.com/Fallenbagel/jellyseerr/assets/98979876/e06a0697-5111-4c2c-844c-a38444e14a7b)

The `Remove from Radarr` button.

**Reply by ezekieldas** [reply]
I think we're crosstalking. But anyhow, the visual helps. That was the right place... but this requires full Admin (which is ok for this setting). 

Testing this...

**Comment by ezekieldas** [comment]
:point_up: Works like a charm. 

BTW, fantastic addition to pipeline; quick, minimal footprint, clear navigation, immediately containerized. I wish I'd implemented this one well before similar options.
